CRM/Core/Smarty.php

```diff
@@ -187,6 +187,19 @@ public function fetch($resource_name, $cache_id = NULL, $compile_id = NULL, $dis
     return $output;
   }
 
+  /**
+   * Ensure these variables are set to make it easier to access them without e-notice.
+   *
+   * @param array $variables
+   */
+  public function ensureVariablesAreAssigned(array $variables): void {
+    foreach ($variables as $variable) {
+      if (!isset($this->get_template_vars()[$variable])) {
+        $this->assign($variable);
```

Comment:
what is the actually value here that is getting assigned?

Reply:
null is the default